pkg/controller/certificaterequests/acme/acme.go

```diff
@@ -218,8 +218,12 @@ func buildOrder(cr *v1.CertificateRequest, csr *x509.CertificateRequest) (*cmacm
 		IPAddresses: ipAddresses,
 	}
 
+	if issuer.GetSpec().ACME.RequestDuration {
+		spec.Duration = cr.Spec.Duration
+	}
```

Comment:
Add a unit test for this...and maybe don't pass the whole issuer in but just the `requestDuration bool` which will make it simpler to unit test.